src/unix/core.c

```diff
@@ -41,6 +41,10 @@
 #include <sys/resource.h> /* getrusage */
 #include <pwd.h>
 
+#if defined(__linux__)
+# include <sched.h>
```

Comment:
This header is available on more OSes, at least on NetBSD. Is there need to restrict it to Linux-only?

Reply:
Not in the long-term, but for now my goal is to get the API worked out with sample implementaions on Linux and Windows.  Additional ports may follow.

Reply:
Please enable it on all OSes unconditionally. (unless there exists a UNIX-like system without this header)

Reply:
Done!